dotnet/src/Functions/Functions.OpenApi/OpenApi/OpenApiDocumentParser.cs

```diff
@@ -378,21 +362,26 @@ internal static List<RestApiSecurityRequirement> CreateRestApiOperationSecurityR
     /// </summary>
     /// <param name="operationId">The operation id.</param>
     /// <param name="parameters">The OpenAPI parameters.</param>
+    /// <param name="logger">The <see cref="ILogger"/> to use for logging.</param>
     /// <returns>The parameters.</returns>
-    private static List<RestApiParameter> CreateRestApiOperationParameters(string operationId, IList<OpenApiParameter> parameters)
+    private static List<RestApiParameter> CreateRestApiOperationParameters(string operationId, IList<OpenApiParameter> parameters, ILogger logger)
```

Comment:
I wonder what prevents us from wrapping the code in the CreateRestApiOperations method in a try-catch block and logging errors in one central place?

Reply:
Good point, updated.